scenarios/chapter1/01_The_Uprooting.cfg

```diff
@@ -540,7 +520,7 @@ This is the story of Kalenz, Landar, and of the elves in the first days of the h
 
         [message]
             speaker=second_unit
-            message= _ "Take that, you orcish scum!"
+            message= _ "Savour thine recompense, beast!"
```

Comment:
I doubt about making battle cries that old sounding. It's also harder to understand (I had to look up it in the dictionary). To my taste, simple rough phrases sound better for this "in-the-heat-of-the-battle" kind of battle cries

Reply:
Yeah, I got a bit carried away. Changed it to the much simpler: `"Too slow, beast!"`